BootstrapKit/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,14 @@
+## Working Group Code of Conduct Template
+
+Every Working Group must have a Code of Conduct. By default,
+all repositories under the Node.js GitHub Organization fall under
+the [Node.js Code of Conduct][], but Working Groups can choose to
+define their own alternative policy. If the Working Group does choose
+it's own Code of Conduct, it must be documented in a `CODE_OF_CONDUCT.md`
+file located in the root of any repository under their stewardship.
```

Comment:
How do you define "bypass" though? That's really hard and you're opening up the language in every lower level CoC to be under a strange bikeshed where people re-interpret the language in the base CoC.

There's really only two ways you can phrase this to be enforcable and sustainable:
* Make the base CoC apply to all WGs and TLPs not matter what, allowing only *additional* terms to be added by independent bodies.
* Allow the CoC to be replaced entirely and if the TSC is upset enough about it they can yank the charter (extreme and doubtful but is quite literally the fallback mechanism we have for all other policies and it seems to be working so far).